第一天学习反馈优化教案

SONDER0521 opened the pull request:
django安装指定为4.2版本
去掉重复让人误会的部分，如创建应用。
更加学员的不解，对不解部分做出解释修改，如data解释

SONDER0521 committed:
modified:   docs/contents/P01.md
	modified:   docs/contents/P02.md
	modified:   docs/contents/P06.md

SONDER0521 committed:
Merge branch 'main' of https://github.com/SONDER0521/sweettalk-django4.2

Joe-2002 reviewed (commented):
![2024-05-14-19733](https://github.com/Joe-2002/sweettalk-django4.2/assets/56438623/0a088a8f-b19e-4c06-a570-dc35d613759b)
P02有图片命名问题，注意校正。

SONDER0521 committed:
modified:   docs/contents/P02.md
	modified:   docs/contents/P03.md
	modified:   docs/contents/P04.md
	modified:   docs/contents/P05.md
	modified:   docs/contents/P06.md
	modified:   docs/contents/P07.md
	modified:   docs/contents/P13.md
	renamed:    docs/contents/assets/image20230814201211-settings.png -> docs/contents/assets/configure-settings.png
	renamed:    docs/contents/assets/image-20230531125638-pq3meyv.png -> docs/contents/assets/data_apps.png

Joe-2002 commented:
good job

SONDER0521 commented:
还增加了软链接
